### ISSUE TITLE
E-mail SPAM detection model using Multinomial Naive Bayes algorithm[EXE]

### ISSUE THREAD

**Issue by darpan-2001** [gimseng/99-ML-Learning-Projects#194]
#### Learning Goals
Naive Bayes algorithm, text feature extraction, sklearn pipeline object.

### Exercise Statement
Create an e-mail spam detection model.

### Prerequisites 
Basic understanding of Naive Bayes algorithm, Python and scikit-learn basics.

### Data source/summary:
Dataset if obtained from Kaggle. Data consists of email messages, already labeled as spam or ham.
Link of dataset: https://www.kaggle.com/code/mfaisalqureshi/email-spam-detection-98-accuracy/data

### (Optional) Suggest/Propose Solutions
I have the solution ready implemented with Multinomial Naive Bayes algorithm,  will be happy to create pull request to include the exercise solution.

**Comment by lucifertrj** [comment]
I will take this one, could you assign me this project @darpan-2001?

**Comment by DhruvSanghvi2002** [comment]
I would love to solve this challenge ,kindly assign me this issue @darpan-2001

**Comment by Adarshh9** [comment]
can you assign me this issue?
